.github/workflows/docker-build.yml

```
@@ -36,14 +36,24 @@ jobs:
+      - name: Extract metadata for Docker (stable release with latest tag)
         id: meta
+        if: ${{ !github.event.release.prerelease }}
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+          tags: |
+            type=ref,event=tag
+            type=raw,value=latest
+
+      - name: Extract metadata for Docker (prerelease)
+        id: meta
```

actionlint (1.7.8)

step ID "meta" duplicates. previously defined at line:40,col:13. step ID must be unique within a job. note that step ID is case insensitive

(id)